components/odh-notebook-controller/controllers/notebook_oauth.go

```diff
@@ -104,6 +112,89 @@ func (r *OpenshiftNotebookReconciler) ReconcileOAuthServiceAccount(notebook *nbv
 			log.Error(err, "Unable to fetch the Service Account")
 			return err
 		}
+
+		log.Info("Before conditional statement")
+
+		// Add view roles to the Service Account if the workbench is DataScience project based
+		if notebook.Namespace != ODHDefaultNameSpace && notebook.Namespace != RHODSDefaultNameSpace {
+
+			log.Info("Inside if statement")
```

Comment:
I'm not sure how these `Info` logs will show in the console. From the brief look these were made for the debugging purpose only? Therefore, shouldn't we move it to the debug level? Or, in case we want to keep them, we should rephrase to something like:

```suggestion
			log.Info("Adding roles to the Service Account for the DataScience project")
```

Reply:
Had to add logs like this as I was trying out a few things to get the PR image built, will remove them once I am done, marking the PR wip for now.